projects/proj-event-cards.md

```diff
@@ -0,0 +1,105 @@
+
+*Note*: For this task, we can utilize Codepen's native features to link pens together.
+
+To include the HTML from another pen, add the pen url, surrounded by three brackets.
+```html
+[[[https://<url to your event card>]]]
+```
+
+You will also need to link to your pen as an external css resource by going to *Settings -> CSS - > Add External Stylesheets/Pens* and adding the URL of your card.
+
+### Browser Testing
+*Note*: This is a big ask. It's not fun. It won't be perfect. You don't have the full resources of a design team. Do the best you can, but don't burn out.
+
+At Sparkbox, we don't have Quality Assurance, so we usually do our own browser testing. If there is interest, we can do a 'bug bash' together, to learn about the device wall and [BrowserStack](https://www.browserstack.com/start).
+
+
+#### Gold Level Support
+In these browsers, behavior & design are strong representations of design intent. A user is able to access all content and functionality as described.
+
+macOS
+- Safari 9+ (latest subversion)
+- Chrome (latest version)
+- Firefox (latest version)
+
+Windows
+- IE 11+
+- Chrome (latest version)
+- Firefox (latest version)
+- Edge 16
+
+Mobile
+- Safari 9+
+- Android 6+
+- Samsung Browser (latest version)
+- Chrome (latest version)
+
+#### Silver Level Support
+In these browsers, behavior & design are acceptable, but may not be as originally intended. A user is still able to access all content and functionality as described.
+
+- IE 10 on Windows
+- Android 5+
+- Safari 7+
+
+#### Bronze Level Support
+In these browsers, behavior & design may not be great, but the content doesn't appear "broken" to the user, either. All content is accessible, and functionality is as described. Additional testing may be required during implementation.
+
+- IE 9 on Windows
+- Android 4.4+
+- Safari 6+
+
+## Week 4 - BEM + IMG
+
+Your homework this week is comprised of a few smaller tasks.
+
+#### BEM
+From this point forward, you'll be expected to use BEM for any CSS and HTML you write. Fork your Grids and refactor all of your classes to use BEM. Then, pair with one of our BEM devs to review your *finished* BEM.
+
+#### SVG or `srcset`
+Spend some time playing with images or SVGs. Fork and update your event card to use either an SVG icon (Next to your address or phone number is nice, or whatever you like) OR update your images to use `srcset`.
+
+#### Personal Websites
+It's about the time in your apprenticeship when you should be looking to find a permanent web development role (if you are interested in that sort of thing). In your career prep sessions, you've hopefully developed skills to help you improve your marketability. Spend some time reworking or redoing your personal websites, cleaning up your git profiles, etc. We're here to help, so pair!  Remember to take advantage of pairing time not only with developers, but also with our wonderful Project Management, Content, and Design teams.
```

Comment:
@silverli, can we get a short blip about _career prep sessions_ somewhere in here that @catheraaine can link to?